addons/web/static/src/scss/utilities_custom_backend.scss

```diff
@@ -5,24 +5,32 @@
 $utilities: map-merge(
     $utilities,
     (
-        // Limit 'text-X' classes generation.
+        // For the other 'text-X' classes.
         // See bootstrap_review_backend.scss
-        "color": (
-            property: color,
-            class: text,
-            local-vars: (
-                "text-opacity": 1
-            ),
-            values: (
-                "body": $body-color,
-                "muted": $text-muted,
-                "reset": inherit,
+        "color": map-merge( 
```

Comment:
Using `map-merge()` will generate Bootstrap's default classes as well, while we want to generate them using `bootstrap_review_backend` instead.